src/wallet/wallet.cpp

```diff
@@ -1982,7 +1982,7 @@ void CWallet::ResendWalletTransactions()
     // that these are our transactions.
     if (GetTime() < nNextResend || !fBroadcastTransactions) return;
     bool fFirst = (nNextResend == 0);
-    nNextResend = GetTime() + GetRand(30 * 60);
+    nNextResend = GetTime() + GetRand(24 * 60 * 60);
```

Comment:
This is not frequency 1/day (as the PR post says). It's a value between 0 and 24 hours, with an average of 12 hours. You probably want to do 12*60*60 + GetRand(12*60*60) or something like that?

Reply:
`12*60*60 + GetRand(12*60*60)` would get an average of 24 hours with a range of 12-36

Reply:
@ajtowns yeah, that's what would be closer to 1/day, right? Not sure what you meant to say.

Reply:
Oh my god. `12*60*60 + GetRand(24*60*60)` was what I was trying to type; min of 12h, max of 36h, average of 24h.

Reply:
Agree, I think we should do what you just suggested.

Reply:
updated.

funny to update the code to match the comment language rather than vice versa 😛 but I don't have a strong preference around 12-36hrs vs 0-24hrs, both have small tradeoffs.